bus/nsq/consumer_test.go

```diff
@@ -134,6 +135,12 @@ func TestConsumer_ConnectLookupds(t *testing.T) {
 	topic := "testtopic"
 	channel := "testchannel"
 
+	// create test topic just in case
+	_, err := http.Post("http://127.0.0.1:4151/topic/create?topic="+topic, "", nil)
+	if err != nil {
+		log.Println(err)
+	}
+
```

Comment:
The problem with the test is that if the topic doesn't exist then the consumer can't connect  to nsqd. To fix this I always make sure the topic is created before verifying that we are connecting to lookupd